### PR TITLE
Adds fancy UI

### PR DESCRIPTION
**Pull request by TheGreyDiamond**
Don't merge yet, but this will add a cool ui.

- [x] Finish landing page

### CODE REVIEW

**Comment by TheGreyDiamond** [comment]
The landing page is done, @XilefTech  please look through his to check if you agree!

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (XilefTech) [inline .vscode/settings.json]: don't ya think this should go into the gitignore? everyone has their own python paths and stuff...

File: templates/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <video autoplay loop muted class="absolute z-10 w-auto min-w-full min-h-full max-w-none">
+          <source src="https://assets.mixkit.co/videos/preview/mixkit-stars-in-space-1610-large.mp4" type="video/mp4" />Your browser does not support the video tag.
+        </video>
+
+      </header>
+      <div class="max-w-lg m-auto text-white">
+        <p class="mb-4">
+          Everyone knows the problem: You are in a hurry and are trying to decipher your WebUntis -
+           and then it happens you make a mistake, you are too late for a lesson, or in a wrong room. This creates pain and agony.
+           This happens because many schools don't setup their WebUntis Instance very good. But <b>UnitFilter</b> is here to help.
```
Comment (XilefTech) [inline templates/index.html]: I'd say "very well" instead of good

**Review by XilefTech — COMMENTED** [review]
seems fine so far... just look at the few points I annotated again pls

File: templates/register.html [hunk 3]
```diff
@@ -1,23 +1,110 @@
...
+<body class="dark:bg-gray-800 h-screen overflow-hidden flex items-center justify-center" onresize="trinagle">
+
+  
+  <div class="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 flex-none flex-col w-4/12">
+    <form action="" method="post">
+    <div class="mb-4">
+      <label class="block text-grey-darker text-sm font-bold mb-2" for="email">
+        E-Mail
+      </label>
+      <input class="shadow appearance-none border rounded w-full py-2 px-3 text-grey-darker" id="email" type="text" placeholder="john.doe@example.com" name="email" value="{{ request.form.email }}">
```
Comment (XilefTech) [inline templates/register.html]: you can make this type email again

File: templates/register.html [hunk 4]
```diff
@@ -1,23 +1,110 @@
...
+
+      <label class="block text-grey-darker text-sm font-bold mb-2" for="class">
+        Class:
+      </label>
+      <input class="shadow appearance-none border border-red rounded w-full py-2 px-3 text-grey-darker mb-3" id="class" name="class" type="text" placeholder="Your Class" value="">
+
+      <label class="block text-grey-darker text-sm font-bold mb-2" for="class_password">
+        Class Password:
+      </label>
+      <input class="shadow appearance-none border border-red rounded w-full py-2 px-3 text-grey-darker mb-3" id="class_password" name="class_password" type="text" placeholder="Your Class password" value="">
```
Comment (XilefTech) [inline templates/register.html]: shouldn't this be type password as well?

**Review by XilefTech — CHANGES_REQUESTED** [review]
found some more things

File: templates/index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+        </p>
+        <p class="mb-12">
+          We help you filter and sort your WebUntis to make it more sane. Displying only the courses you <b>really</b> care about.
```
Comment (XilefTech) [inline templates/index.html:46]: typo in displaying ;)

also you can make these paragraphs a bit bigger in text size

**Review by XilefTech — APPROVED** [review]

**Pull request merged**